tests/src/Functional/ContactsDashboardTest.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
-namespace Drupal\contacts\Tests;
+namespace Drupal\Tests\contacts\Functional;
 
-use Drupal\simpletest\WebTestBase;
+use Drupal\Tests\BrowserTestBase;
 
 /**
  * Tests CRM fields and views.
  *
  * @group contacts
  */
-class ContactsDashboardTest extends WebTestBase {
+class ContactsDashboardTest extends BrowserTestBase {
 
   public static $modules = ['user', 'contacts', 'views'];
```

Comment:
FIxing that in a follow up issue:
https://github.com/FreelyGive/contacts/pull/11